File: Semester 2/Research Project/2020_AON01/Stage_API/Stage_API.Domain/Classes/Bedrijf.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Stage_API.Domain.Classes
{
    public class Bedrijf : User
    {
        //Algemeen
        public string Adres { get; set; }
        public string Postcode { get; set; }
        public string Gemeente { get; set; }
        //Totaal aantal medewerkers in het bedrijf
        public int AantalMedewerkers { get; set; }
        public int AantalITMedewerkers { get; set; }
        //Aantal personen die de student technisch kunnen begeleiden bij het uitwerken van zijn stageopdracht  
        public int AantalBegeleidendeMedewerkers { get; set; }
        //Contactpersoon
        public Contactpersoon Contactpersoon { get; set; }
        [IgnoreDataMember]
        public int ContactpersoonId { get; set; }
        //Bedrijfspromotor
        public Bedrijfspromotor Bedrijfspromotor { get; set; }
        [IgnoreDataMember]
        public int BedrijfspromotorId { get; set; }
        public ICollection<Stagevoorstel> Stagevoorstellen { get; set; }
    }
}

File: Semester 2/Research Project/2020_AON01/Stage-Frontend/src/app/classes/bedrijf.ts
import {Contactpersoon} from './contactpersoon';
import {Bedrijfspromotor} from './bedrijfspromotor';
import {User} from './user';


export class Bedrijf extends User {
  adres: string;
  postcode: string;
  gemeente: string;
  aantalMedewerkers: number;
  aantalITMedewerkers: number;
  aantalBegeleidendeMedewerkers: number;
  contactpersoon: Contactpersoon;
  bedrijfspromotor: Bedrijfspromotor;
}
